### PR TITLE
Adopt icon font in problems and peek error

### LINKED ISSUE #74921: Theme error/warning/info icons
Theme error/warning/info icons

### PR DESCRIPTION
**Pull request by miguelsolorio**
Related #78889 and fixes #74921

This adopts the icon font in the problems and peek error views:

image

I've also created a few color tokens:
- `problemsErrorIcon.foreground` which inherits from `editorError.foreground`
- `problemsWarningIcon.foreground` which inherits from `editorWarning.foreground`
- `problemsInfoIcon.foreground` which inherits from `editorInfo.foreground`

I've also made the notification icons (`notificationsErrorIcon.foreground` etc.) inherit from the same corresponding editor foreground colors from above so that they're all updated like:

image

cc @bpasero for the notification icons inheritance

### CODE REVIEW

File: src/vs/workbench/contrib/extensions/browser/extensionsViews.ts [hunk 1]
```diff
@@ -708,10 +708,10 @@ export class ExtensionsListView extends ViewletPanel {
 				if (count === 0 && this.isBodyVisible()) {
 					if (error) {
 						if (error instanceof ExtensionListViewWarning) {
-							this.bodyTemplate.messageSeverityIcon.className = SeverityIcon.className(Severity.Warning);
+							this.bodyTemplate.messageSeverityIcon.className = 'codicon ' + SeverityIcon.className(Severity.Warning);
```
Comment (bpasero) [inline src/vs/workbench/contrib/extensions/browser/extensionsViews.ts]: Can be written more elegant as \`codicon ${SeverityIcon.className(Severity.Warning)}\`

**Comment by aeschli** [comment]
Looks good to me!

**Comment by sandy081** [comment]
LGTM

**Comment by rediffusion** [comment]
Hi all. I googled some ℹ️ information about a few features with no result:

```
editorWarning.foreground
editorWarning.border
editorHint.foreground
editorHint.border
problemsWarningIcon.foreground
```

 In order to understand, please provide some screenshot, screencast or short video❗ 

P.S. ··· documentation on [official site](https://code.visualstudio.com/api/references/theme-color) sucks, thanks❗

Best Regards✌️

**Pull request merged**